src/vscode-dts/vscode.d.ts

```diff
@@ -8800,7 +8800,7 @@ declare module 'vscode' {
 		 * The problem matchers attached to the task. Defaults to an empty
 		 * array.
 		 */
-		problemMatchers: string[];
+		problemMatchers: string | string[];
```

Comment:
I don't think we can make this change because task can be "read" via `task.fetchTasks()`. This means there can be consumers of `Task#problemMatchers` which expect this to be an array and not a single string

Reply:
Ok, makes sense. The change in the `extHostTask.ts` file should be sufficient.